CMakeLists.txt

```diff
@@ -61,6 +61,12 @@ set(CMAKE_CXX_VISIBILITY_PRESET hidden)
 set(CMAKE_VISIBILITY_INLINES_HIDDEN TRUE)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/bin)
 
+execute_process(
+  COMMAND conan create . scipp/stable
+  WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}/.conan-recipes/llnl-units"
+                    COMMAND_ECHO STDOUT
+)
+
```

Comment:
Related to above: Using the same compiler version in CMake as in my conan profile does not produce the above error. But `units` gets compiled twice. There is probably still a mismatch between the profile and settings used by CMake.
I don't know if there is a way to not compile during `conan create` but only export `conanfile.py`. This would be sufficient because here, there is no need to compile / test the package in addition to the regular scipp build.